workitem/typegroup/typegroup.go

```diff
@@ -68,7 +68,7 @@ var Requirements0 = WorkItemTypeGroup{
 var Execution0 = WorkItemTypeGroup{
 	Group: GroupExecution,
 	Level: []int{2, 0},
-	Name:  "Execution",
+	Name:  "Iterations",
```

Comment:
done in https://github.com/fabric8-services/fabric8-wit/pull/1580/commits/329e99dfae740875218141a6ad2bb3b1e493a643